app/controllers/publishers_controller.rb

```diff
@@ -110,14 +110,12 @@ def protect
 
   def ensure_email
     token = params[:token]
-    confirm_email = params[:confirm_email]
-
     publisher = Publisher.find(params[:id])
 
-    if PublisherTokenAuthenticator.new(publisher: publisher, token: token, confirm_email: confirm_email).perform(consume: false)
+    if token && PublisherTokenAuthenticator.new(publisher: publisher, token: token).perform(consume: false)
       @publisher = publisher
     else
-      flash[:alert] = t(".token_invalid")
+      flash[:alert] = t("shared.error")
```

Comment:
I had to change this to a shared error because it was missing the JA translation and we don't have time for that.